src/events/PickFacultyEvent.ts

```diff
@@ -1,5 +1,13 @@
 import {IEvent, IChoice, StatChanges} from "./core";
 
+import ARTS from "../assets/ARTS.png";
+import APSC from "../assets/APSC.png";
+import COMM from "../assets/COMM.png";
+import FORE from "../assets/FORE.png";
+import KIN from "../assets/KIN.png";
+import LFSlogo from "../assets/LFS.png";
+import SCIE from "../assets/SCIE.png";
```

Comment:
I... didn't know you could do that....

Reply:
Could we just leave this in html? It's just a square with some text in the middle

Reply:
What do you mean by leaving this in html?

Reply:
instead of importing it as an png?

Reply:
Like the quality drops a little when it is exported as a png

Reply:
oh, as in leaving it in HTML SVG? if that's the case, sure. export it as SVG.

Reply:
No like literally just code them as HTML elements. Like replace it with <div class="faculty-icon">SCIE</div> for example.

Reply:
The only thing we are changing is the color code and the text in the center. Shouldn't be too hard to implement, no?

Reply:
Good point.

Reply:
I tried doing this with the framework we have, it requires some hard coding, and the code would be a bit ugly if you still want that i can flip it back

Reply:
Personally unsure of how this should be handled - I see value in keeping these images in case we choose to have fancier images for the logos of each faculty in the future. Seeing as though we haven't decided on whether this is the case, I'm ok leaving this as is and refactoring later if we decide that we want to display all this with pure HTML, especially since we need this merged soon.